src/BlazorStrap/util/StringExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Text;
+
+namespace BlazorStrap.util
+{
+	public static class StringExtensions
+	{
+		public static string RemoveUnneededSpaces(this string source)
+		{
+			if (string.IsNullOrEmpty(source))
+				return string.Empty;
+
+			bool bFound = false;
```

Comment:
Would something like this work better? Regex.Replace(input, @"\s+", " ").Trim(); - this will strip all dup white spaces from the string. Still need to call Trim to remove any leading and ending spaces